README.md

```diff
@@ -71,6 +71,27 @@ Error: Type error on call: [{bytes,<<240,2,...>>}] is not of type [{bytes,32}]
 Revert: require failed
 ```
 
+## Events
+
+Example:
+```javascript
+const {Encoder} = require('@aeternity/aepp-calldata')
+const ACI = require('./Test.json')
+
+const encoder = new Encoder(ACI)
+
+ const data = encoder.decodeEvent('Test', 'cb_dHJpZ2dlcmVk1FYuYA==', [
```

Comment:
it requires a contract name, so in my implementation of remote contract events decoding I have to try/catch all available contracts:
https://github.com/aeternity/aepp-sdk-js/pull/1361/files#diff-3eb56adde236d7d371c8e12d1145d20f88e1d6b94cb05631ed2d7430ff658e9dR259-R278
I don't know how to make this decoding better

Reply:
I don't get it ? To decode an event we need the type info unfortunately. The event type info is scoped per contract for obvious reasons. So one needs to know the contract name to decode its events.

Reply:
> So one needs to know the contract name to decode its events, assuming multiple contracts per ACI.

If event names are unique for all contracts in ACI we can find the corresponding contract by ourselves (implemented in my PR). Otherwise probably the best is to ask developer to provide a mapping of the contract address to its name.

Why the first topic is not a hash of the contract name concatenated with the event name?

Reply:
> If event names are unique for all contracts in ACI

But that's not the case.

Reply:
> Why the first topic is not a hash of the contract name concatenated with the event name?

Because there is no contract name on the blockchain ?

Reply:
> it requires a contract name, so in my implementation of remote contract events decoding I have to try/catch all available contracts

While I don't want to interfere with the SDK implementation, I wouldn't implement it that way. The SDK already have the ACI and can check if a given event is supported for a given contract.

In general I believe that exception driven development is bad practice, not only because of performance (yeah, it carry a backtrace usually), but also for readability.